tests/by-util/test_install.rs

```diff
@@ -1808,3 +1808,161 @@ fn test_install_symlink_same_file() {
             "'{target_dir}/{file}' and '{target_link}/{file}' are the same file"
         ));
 }
+
+#[test]
+fn test_install_no_target_directory_failing_cannot_overwrite() {
+    let scene = TestScenario::new(util_name!());
+    let at = &scene.fixtures;
+    let file = "file";
+    let dir = "dir";
+
+    at.touch(file);
+    at.mkdir(dir);
+    scene
+        .ucmd()
+        .arg("-T")
+        .arg(file)
+        .arg(dir)
+        .fails()
+        .stderr_contains("cannot overwrite directory 'dir' with non-directory");
+
+    assert!(!at.dir_exists("dir/file"));
+}
+
+#[test]
+fn test_install_no_target_directory_failing_omitting_directory() {
+    let scene = TestScenario::new(util_name!());
+    let at = &scene.fixtures;
+    let dir1 = "dir1";
+    let dir3 = "dir3";
+
+    at.mkdir(dir1);
+    at.mkdir(dir3);
+    scene
+        .ucmd()
+        .arg("-T")
+        .arg(dir1)
+        .arg(dir3)
+        .fails()
+        .stderr_contains("omitting directory 'dir1'");
+
+    assert!(!at.dir_exists("dir/file"));
```

Comment:
This `assert` doesn't make sense to me as this test function is about `dir1` and `dir3`.